Configuration/PyReleaseValidation/python/upgradeWorkflowComponents.py

```diff
@@ -78,7 +80,7 @@
 }
 numWFSkip=200
 # temporary measure to keep other WF numbers the same
-numWFConflict = [[13200,13600],[20000,23200],[23600,28200],[28600,31400],[31800,32200],[32600,34600],[50000,51000]]
+numWFConflict = [[20000,23200],[23600,28200],[28600,31400],[31800,32200],[32600,34600],[50000,51000]]
```

Comment:
This removal was not in the backported PR: what's the purpose of removing this here?
They were added by #40138 for 12_4_X with the postEE scenarios, and I think they should stay

Reply:
This is an intention. Since the postEE PR was merged first in 12_4, the `[13200,13600]` was added to add exception on this range. This range is used for Run-3 FastSim. With this setting, the workflow ID for postEE in master and 12_4 are the same. 

When this FastSim Run-3 PR comes, workflows are available. So we don't need to keep the `numWFConflict` anymore.